Sources/Sentry/SentryHub.m

```diff
@@ -93,6 +96,11 @@ - (void)startSession
         }
         _session = [[SentrySession alloc] initWithReleaseName:options.releaseName];
 
+        if (_errorsBeforeSession > 0) {
```

Comment:
`h`: We should only do this if `options.enableAutoSessionTracking == true`. We would mess up the statistics if the user switches to manual mode.

Reply:
Seems a bit weird to me. The user captured an error, and they started a session.. so no matter if `enableAutoSessionTracking` is on or off, that session's error count should be increased I would think?

But I made the change and added a test for it.